### ISSUE TITLE
How to log access logs to file any configuration?

### ISSUE THREAD

**Issue by grku2opia** [reactor/reactor-netty#301]

**Comment by violetagg** [comment]
Hi,

There is no access log available at the moment.
Mark this as enhancement.

Regards,
Violeta

**Comment by spencergibb** [comment]
Users of spring-cloud-gateway are asking for this.

**Comment by chsi13** [comment]
What about this feature ?

**Comment by tillmannheigel** [comment]
+1

**Comment by imet** [comment]
+1

**Comment by stalelabs** [comment]
+1

**Comment by jayendra155** [comment]
+1

**Comment by bruno-roussel** [comment]
+1

**Comment by the-fine** [comment]
+1

**Comment by rgeoroceanu** [comment]
+1

**Comment by violetagg** [comment]
Fixed with e26831e
Only `CLF` is supported for the moment.
`-Dreactor.netty.http.server.accessLogEnabled=true` can be used to enable access log (by default it is disabled)
The configuration below (logback) or similar (other logging frameworks) can be used to have a separate access log file
```
    <appender name="accessLog" class="ch.qos.logback.core.FileAppender">
        <file>access_log.log</file>
        <encoder>
            <pattern>%msg%n</pattern>
        </encoder>
    </appender>
    <appender name="async" class="ch.qos.logback.classic.AsyncAppender">
        <appender-ref ref="accessLog" />
    </appender>

    <logger name="reactor.netty.http.server.AccessLog" level="INFO" additivity="false">
        <appender-ref ref="async"/>
    </logger>
```
